packages/alt-ergo/alt-ergo.0.95.2/opam

```diff
@@ -1,24 +1,34 @@
+(******************************************************************************
+* Please, do not modify this file without contacting the maintainers
+* to discuss the reasons of the modification.
+* In particular, dev-repo is not provided because it's private.
+******************************************************************************)
```

Comment:
It's sometimes annoying when people modify metadata of opam packages without referring to developers (in particular, when they put wrong dependencies or wrong dev-repos).

I don't see other alternatives since "everyone" can modify a package in OPAM